bundles/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/utilities/ICloudTextTranslator.java

```diff
@@ -35,12 +36,12 @@ public ICloudTextTranslator(Bundle bundle, TranslationProvider i18nProvider, Loc
         this.localeProvider = localeProvider;
     }
 
-    public String getText(String key, Object... arguments) {
+    public String getText(@Nullable String key, Object... arguments) {
```

Comment:
not sure here (didn't check the whole code): is there any useful case where `key == null` is possible? if not, check for null where this is instantiated.